scripts/qa_crowbarsetup.sh

```diff
@@ -2431,6 +2442,14 @@ EOF
         fi
     fi
 
+    if [ -n "$wantdocker" ]; then
```

Comment:
Same comment here.  Alarm bells should be ringing every time you have to add code to a long function (or method).